### PR TITLE
Fix i18n for emails

### LINKED ISSUE #534: Emails not being translated
(I'm currently just assuming that I've triaged this correctly)

In https://github.com/sillsdev/languageforge-lexbox/pull/516 I moved out i18n initialisation from the hooks to the root layout (because that made it easier/possible to pick the correct locale).

The root layout doesn't get used when we generate emails, because we only render individual components. So, now i18n never gets initialised and emails look like this:
image

### PR DESCRIPTION
**Pull request by myieye**
Resolves #534 

Init i18n for emails and send Accept-Language header to svelte when rendering emails.

I also opened a [tiny PR upstream](https://github.com/cibernox/precompile-intl-runtime/pull/46) that would let us tidy up the .NET code a tad.

### CODE REVIEW

**Review by hahn-kev — CHANGES_REQUESTED** [review]
I didn't see a problem testing locally with English but maybe that's just the default.

File: backend/LexBoxApi/Services/EmailService.cs [hunk 1]
```diff
@@ -88,7 +88,7 @@ public async Task SendCreateProjectRequestEmail(LexAuthUser user, CreateProjectI
         var email = new MimeMessage();
         email.To.Add(new MailboxAddress("Admin", _emailConfig.CreateProjectEmailDestination));
         await RenderEmail(email,
-            new CreateProjectRequestEmail("Admin", new CreateProjectRequestUser(user.Name, user.Email), projectInput));
+            new CreateProjectRequestEmail("Admin", new CreateProjectRequestUser(user.Name, user.Email), projectInput), user.Locale);
```
[the remark this answers is not in the document]
Comment (myieye) [reply backend/LexBoxApi/Services/EmailService.cs:91]: 😆!! Good catch!!

[the remark this answers is not in the document]
Comment (myieye) [reply backend/LexBoxApi/Services/EmailService.cs]: Fixed it

**Comment by myieye** [comment]
This PR is now on hold. See #545.

**Comment by myieye** [comment]
Will be covered in #545

**Pull request closed without merging**